truss/templates/server/common/logging.py

```diff
@@ -13,6 +13,15 @@
 )
 
 
+class HealthCheckFilter(logging.Filter):
+    def filter(self, record: logging.LogRecord) -> bool:
+        # for any health check endpoints, lets skip logging
+        return (
+            record.getMessage().find("/") == -1
+            and record.getMessage().find("/v1/models/model") == -1
```

Comment:
It would be good to move the healthcheck endpoints to constants somewhere. I feel this `/v1/models/model` is too verbose and we may want to change this in future, so would be good to have it consolidated as constant.